tests/test-defs.cmake

```diff
@@ -18,7 +18,7 @@ set(ISOLATION_TEST_SCHEDULE ${CMAKE_CURRENT_BINARY_DIR}/isolation_test_schedule)
 
 set(PG_REGRESS_OPTS_BASE
   --host=${TEST_PGHOST}
-  --load-language=plpgsql
+  --load-extension=plpgsql
```

Comment:
is this pg-version specific?

Reply:
Yes. https://github.com/postgres/postgres/commit/751c63cea0befe292fafd9a4a1f16dfa2f1746f9 It's equivalent to --load-extension and this option is remove in newer version of pg.